src/components/UI/Card.js

```diff
@@ -1,12 +1,36 @@
-import React from "react";
+import classNames from "classnames";
+import React, {useState} from "react";
 import "./Card.css";
 
-function Card(props) {
+function Card({item}) {
+  // let [classNames, setClassNames] = useState("card");
+  const [isChecked, setChecked] = useState(false);
+
+  const clickHandler = () => {
+    // if (classNames === "card") {
+    //   setClassNames("card card_selected");
+    // } else {
+    //   setClassNames("card");
+    // }
+    setChecked(!isChecked)
+  };
```

Comment:
I would advice to:
- use ES6 arrow functions syntax - will allow to rewrite the same in one line
`const clickHandler = () => setChecked(!isChecked);`
- remove the commented code - to not store artifacts, these code lines were committed before so a part of the history anyway.